### ISSUE TITLE
Error when adding new event type

### ISSUE THREAD

**Issue by piotrpog** [verbb/Events#150]
### Describe the bug

I installed verbb events, tried to add event type and got this error. Page got redirected to this url:

```
events/purchased-tickets?site=default
```
And this error showed up.

```
SQLSTATE[42S02]: Base table or view not found: 1146 Table 'db.content' doesn't exist
The SQL being executed was: SELECT `elements`.`id`, `purchasedtickets`.`eventId`, `content`.`title`, `eventtypes`.`name` AS `eventTypeName`
FROM `elements` `elements`
INNER JOIN `content` `content` ON `content`.`elementId` = `elements`.`id`
INNER JOIN `events_purchasedtickets` `purchasedtickets` ON `purchasedtickets`.`eventId` = `elements`.`id`
INNER JOIN `events_events` `events` ON `purchasedtickets`.`eventId` = `events`.`id`
INNER JOIN `events_eventtypes` `eventtypes` ON `events`.`typeId` = `eventtypes`.`id`
GROUP BY `typeId`, `eventId`, `eventTypeName`, `title`, `elements`.`id`
```

Even type seem to be added to the system besides this error.

### Steps to reproduce

1. Install plugin
2. Try adding event type

### Craft CMS version

5.2.5

### Plugin version

3.0.0-beta.2

### Multi-site?

No

**Comment by engram-design** [comment]
Yep, we're actively working on the beta at the moment with some major changes, there will likely be errors for a bit.

**Comment by engram-design** [comment]
Fixed in [3.0.0-beta.3](https://github.com/verbb/events/releases/tag/3.0.0-beta.3)
